kubecost-sub-account-permissions.yaml

```
@@ -68,4 +68,5 @@ Resources:
           - s3:HeadObject
           - s3:List*
           - s3:Get*
-          Resource: "*"
+          Resource:
+          - !Sub 'arn:aws:s3:::${SpotDataFeedBucketName}*'
```

A `*` without a prefacing `/` is a wild card on the name, not the objects.
Though we may need to add a second resource for just the bucket.

```suggestion
          - !Sub 'arn:aws:s3:::${SpotDataFeedBucketName}/*'
```